pywxclient/core/client.py

```diff
@@ -124,6 +124,13 @@ def get_authorize_url(self):
         self._uuid = uuid
         return self._api_cls.get_qrcode_url(self.session, uuid)
 
+    def open_authorize_url(self):
+        """Open WeChat authorization url in system-default browser."""
+        import webbrowser       # stdlib
```

Comment:
It's more standardized to import system standard module in the file header as described at [pep8-imports](https://www.python.org/dev/peps/pep-0008/#imports). And following this rule, it's more friendly to other contributors.

Reply:
Ah, I was mainly considering that you didn't add this function in the first place, and putting imports inside the function defers the memory penalty until someone actually uses `open_authorize_url` itself. Fixed now!